src/main/java/com/ioet/bpm/skills/domain/Skill.java

```diff
@@ -1,36 +1,36 @@
 package com.ioet.bpm.skills.domain;
 
-import com.amazonaws.services.dynamodbv2.datamodeling.*;
-import com.ioet.bpm.skills.domain.converters.CategoryConverter;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAutoGeneratedKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
 import lombok.Data;
 import lombok.NoArgsConstructor;
-
-import javax.validation.constraints.NotBlank;
+import org.springframework.data.annotation.Id;
 
 @NoArgsConstructor
 @Data
 @DynamoDBTable(tableName = "skills_skill")
 public class Skill {
 
+    @Id
     @DynamoDBHashKey
     @DynamoDBAutoGeneratedKey
     private String id;
 
-    @NotBlank
```

Comment:
Do you want skills with a null name?

Reply:
This is the same in master, should i change it anyways?